packages/test-utils/src/ConditionalTest.ts

```diff
@@ -5,9 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-/* eslint-disable jest/no-focused-tests */
```

Comment:
this is weird. But I think it'd work if imported from `@jest/globals`, so probably fine 🙂